### DISCUSSION TITLE
Should IF stake their own and unclaimed $IOTA's

### DISCUSSION THREAD

**Discussion by Tombaaa** [iota-community/Shimmer-Governance#11] [General]
Did not read anything about IF plans to:

a) stake their remaining IOTA's donated to them at ICO (5%) of all tokens at that time 
b) stake the unclaimed IOTA token (65Ti) => found out in the meantime, that they can not be staked, as IF does not have private keys for those tokens

What is your opinion? Shall they stake them and get $SMR?

**Comment by 8bn-io** [comment]
@Tombaaa Good question. I´m also curious if the IF plans to stake the 65Ti. 
I think the IF should stake and get $SMR tokens. Those $SMR tokens would later become part of the IOTA community DAO treasury. Thereby IOTA and Shimmer community would have a stake in the success of one another.

**Reply by 8bn-io** [reply]
@Tombaaa I asked in Discord and I quote Holger / Phylo [IF]: Nope, that is not possible. The Treasury tokens are not accessible, so they can't be moved or do anything until a hardfork releases them.

**Reply by Tombaaa** [reply]
this is true for the unclaimed 65 Ti, as nobody has the private keys, however what about their own tokens, they must have the PK's there

**Reply by antonionardella** [reply]
Are we talking about IF members or the custodial that holds the IF treasury?

**Reply by Tombaaa** [reply]
IF members have their own PK's and I assume, they will stake their tokens.
What I was talking about was the custodial that holds the IF treasury e.g. the remaining tokens if received as Donation from the Crowd Sales.

**Comment by Phyloiota** [comment]
Hey thanks for the input, but sorry, this has nothing to do with the Shimmer NFT launch discussions, so I am closing this topic here to make it not so confusing for others to follow the topics.
